src/test/rpc/Book_test.cpp

```diff
@@ -800,6 +800,460 @@ class Book_test : public beast::unit_test::suite
             books)[jss::status] == "success");
     }
 
+    void testTrackOffers(void)
+    {
```

Comment:
ok - I as sticking with the naming convention for existing test cases in this file...is it worth updating all of them ?